dali/pipeline/operators/python_function/python_function.cc

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/python_function/python_function.h"
+#include <vector>
+
+namespace dali {
+
+DALI_SCHEMA(PythonFunction)
+        .DocStr(R"code(Executes python function that consumes and produces single numpy array.)code")
+        .NumInput(1)
+        .NumOutput(1)
+        .AllowMultipleInputSets()
+        .AddArg("function_id",
+                R"code(Id of the python function.)code",
```

Comment:
This info is kinda vague. It's about `id(func)`, right? Let's put it there explicitly